packages/gafl-webapp-service/src/pages/order-complete/order-complete/__tests__/order-complete.spec.js

```diff
@@ -29,23 +29,29 @@ jest.mock('../../../../constants.js', () => ({
 jest.mock('../../../../processors/price-display.js')
 jest.mock('@defra-fish/business-rules-lib')
 
+const getSampleLicensee = ({
+  postalFulfilment = false,
+  preferredMethodOfConfirmation = 'phone',
+  preferredMethodOfReminder = 'phone'
```

Comment:
Is `phone` valid or should it be `Text`? Some inconsistencies here

Reply:
updated to phone so different for tests